### PR TITLE
[artifactory][artifactory-ha]Added condition to disable the  volumeMounts of migration scripts.

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:
Added condition to disable the  volumeMounts of migration scripts.

### CODE REVIEW

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [10.0.15] - Aug 4, 2020
```
Comment (danielezer) [inline stable/artifactory-ha/CHANGELOG.md:4]: Please add this to the README

Comment (danielezer) [inline stable/artifactory-ha/CHANGELOG.md:4]: Also, the version here is wrong

Comment (chukka) [reply stable/artifactory-ha/CHANGELOG.md:4]: Fixed

Comment (chukka) [reply stable/artifactory-ha/CHANGELOG.md:4]: we already have it  - https://github.com/jfrog/charts/blob/master/stable/artifactory/README.md#artifactory-upgrade-from-6x-to-7x-app-version

**Pull request closed without merging**